crates/runtime/src/dialogue.rs

```diff
@@ -183,27 +189,29 @@ impl Dialogue {
         &mut self.vm.library
     }
 
-    pub fn with_new_program(mut self, program: Program) -> Self {
-        self.set_program(program);
-        self
-    }
-
-    pub fn with_additional_program(mut self, program: Program) -> Self {
-        self.add_program(program);
-        self
+    /// The object that provides access to storing and retrieving the values of variables.
+    /// Be aware that accessing this object will block [`Dialogue::continue_`] and vice versa, so try to not cause a deadlock.
```

Comment:
this will be a footgun :) But I don't know if there's a better way?

Reply:
Don't think so? 🤔